src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const declarations = sourceString.trim().split(';');
+
+  declarations.forEach((declaration) => {
```

Comment:
There is no need to make a copy of the 'declarations' array before iterating over it with 'forEach'. The 'trim' and 'split' methods do not mutate the original 'sourceString', so you can iterate directly without concerns of mutation. [CODE KNOWLEDGE]